interop/hobbits/README.md

```diff
@@ -0,0 +1,71 @@
+# Initial Interop
+
+For the initial client interop, clients will utilize a simplified networking implementation and wire protocol. For more information please review the hobbits specification [here](https://github.com/deltap2p/hobbits).
+
+### Prereqs
+
+- targeting the frozen spec (v0.8)
+- all clients must pass reference tests
+- all clients must pass [hobbits integration tests](./hobbits-integration-tests.md)
+
+- all wire protocol implementations must pass [hobbits conformance tests](https://github.com/jrhea/eth2.0-pm/blob/patch-2/interop/hobbits-conformance-tests.md)
+
+## Stage One: Coordinated Start
+
+Clients simulate a coordinated genesis event and gossip blocks/attestations for each slot.  This stage demonstrates the ability of clients to finalize the chain starting from `Eth2Genesis`.
+
+### Prereqs
+
+- Ability to read in the N number of deposits required to trigger the `Eth2Genesis` log
+- Ability to gossip blocks
+- Static peering
+- TCP transport
+
+### Procedure
+
+For the initial round: 
+1. Clients will pair off in teams of two.
+2. `Eth2Genesis` will be triggered on startup
+3. Once the clients can finalize the chain, then they pick another client pair to interop with.  
+
+Repeat (1) & (2) until all clients are talking.
+
+>**Round 1:** 2 nodes (one per client) with 16 validators/client  
+>**Round 2:** 4 nodes (one per client) with 8 validators/client  
+>**Round 3:** 8 nodes (one per client) with 4 validators/client
+
+### Goal
+
+Clients can declare success at each round after 3 epochs where:
```

Comment:
I would add a "stretch" goal of having run for ~15 minutes without error and continual finalization or something. This seems to be an implied pre-req for "stage two"

Reply:
ya....good idea